tic-tac-toe-app/components/board.jsx

```diff
@@ -0,0 +1,72 @@
+const container = document.querySelector('.container')
+const game = new Game()
+
+const Board = () => {
+  const [box1, setBox1] = React.useState("")
+  const [box2, setBox2] = React.useState("")
+  const [box3, setBox3] = React.useState("")
+  const [box4, setBox4] = React.useState("")
+  const [box5, setBox5] = React.useState("")
+  const [box6, setBox6] = React.useState("")
+  const [box7, setBox7] = React.useState("")
+  const [box8, setBox8] = React.useState("")
+  const [box9, setBox9] = React.useState("")
```

Comment:
Setting state to an empty string nine times for each box is a valid approach. To make your code more even easier to reason about, consider refactoring so that the state of all boxes is set to an array.

Reply:
Oddly enough...I had no idea I could do that. Thank you!